addOns/spider/src/main/java/org/zaproxy/addon/spider/parser/SpiderHtmlParser.java

```diff
@@ -367,6 +373,30 @@ private boolean parseSource(HttpMessage message, Source source, int depth, Strin
             resourcesfound |= processAttributeElement(message, depth, baseURL, el, "background");
         }
 
+        // Process Blockquote elements
+        elements = source.getAllElements(HTMLElementName.BLOCKQUOTE);
+        for (Element el : elements) {
+            resourcesfound |= processAttributeElement(message, depth, baseURL, el, "cite");
+        }
+
+        // Process Param elements
+        elements = source.getAllElements(HTMLElementName.PARAM);
+        for (Element el : elements) {
+            resourcesfound |= processAttributeElement(message, depth, baseURL, el, "value");
+        }
+
+        // Process Base element
+        elements = source.getAllElements(HTMLElementName.BASE);
+        for (Element el : elements) {
```

Comment:
Move to the existing BASE processing, which can be tweaked to just "process" once.

Reply:
Moved. Since base only occurs once per page, should I process only the first occurrence? But then it won't detect all the different cases in the unit test.

Reply:
It's fine to process all of them, we want to find the most possible.